x-pack/plugins/secops/public/components/formatted_duration/helpers.test.ts

```diff
@@ -0,0 +1,401 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  getFormattedDurationString,
+  getHumanizedDuration,
+  ONE_DAY,
+  ONE_HOUR,
+  ONE_MILLISECOND_AS_NANOSECONDS,
+  ONE_MINUTE,
+  ONE_MONTH,
+  ONE_SECOND,
+  ONE_YEAR,
+} from './helpers';
+
+import * as i18n from './translations';
+
+describe('FormattedDurationHelpers', () => {
+  describe('#getFormattedDurationString', () => {
+    test('it returns a placeholder when the input is undefined', () => {
+      expect(getFormattedDurationString(undefined)).toEqual('--');
```

Comment:
To reduce the number of times we use `--` in all places, tests included that need to do an equal compare will use the one spot of:

```ts
expect(getFormattedDurationString(undefined)).toEqual(getEmptyValue());
```

Then later if we change that to a different set of characters you won't have to change out all these tests.

Keeps it dry